python/mxnet/optimizer.py

```diff
@@ -649,6 +657,52 @@ def update(self, index, weight, grad, state):
         # update weight
         weight[:] -= current_delta + wd * weight
 
+<<<<<<< HEAD
```

Comment:
I find that these conflict lines are still here... Could you edit them? I can merge once the checks are passed.